pkgs/by-name/bl/blast-ugjka/package.nix

```diff
@@ -0,0 +1,41 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  makeBinaryWrapper,
+  nix-update-script,
+  pulseaudio,
+}:
+buildGoModule rec {
+  pname = "blast-ugjka";
+  version = "0.7.0";
+
+  src = fetchFromGitHub {
+    owner = "ugjka";
+    repo = "blast";
+    rev = "v${version}";
+    hash = "sha256-yMwMG0y2ehq2dBMlv9hF+i0TgmMjW3ojBVGiqEUSrhU=";
+  };
+
+  vendorHash = "sha256-yPwLilMiDR1aSeuk8AEmuYPsHPRWqiByGLwgkdI5t+s=";
+
+  nativeBuildInputs = [
+    makeBinaryWrapper
+  ];
+
+  postInstall = ''
```

Comment:
`makerel` is also installed, I don't think it is supposed to be

Reply:
fixed so that makerel is not installed (nor built). good catch! and thanks for your reviews :)